eos/form-factors/mesonic-impl.hh

```diff
@@ -590,7 +610,7 @@ namespace eos
 
             static std::string _par_name(const std::string & ff_name)
             {
-                return std::string("B->K^*::alpha^") + ff_name + std::string("@BSZ2015");
+                return std::string(Process_::label) + std::string("::alpha^") + ff_name + std::string("@BSZ2015");
```

Comment:
@akokulu7 Would you be so kind to split this change into a separate earlier commit? This is a true bug fix, even though it did not affect anything before now.